### PR TITLE
RELENG-791 Add pushdeb pool configs

### PR DESCRIPTION
**Pull request by hneiva**

### CODE REVIEW

**Review by escapewindow — CHANGES_REQUESTED** [review]
Looks good, but let's add the level 1 pool in prod as well.

Also, aiui k8s-autoscale PRs should only land after the pools have been created. So we'll need SOPS, cloudops-infra, scriptworker-scripts all set up and deployed to the `production` and `dev` branches at least once, and then we can land this PR.

File: configs/relengworker-prod/config.yml [hunk 1]
```diff
@@ -827,6 +827,20 @@ worker_types:
         capacity_ratio: 1.0
         min_replicas: 0
 
+  - worker_type: mozillavpn-3-pushdeb
```
[the remark this answers is not in the document]
Comment (hneiva) [reply configs/relengworker-prod/config.yml:830]: Maybe have level 1 on nonprod config and level 3 on prod?
Would we be able to use nonprod 1 for staging?

[the remark this answers is not in the document]
Comment (escapewindow) [reply configs/relengworker-prod/config.yml:844]: nonprod level 1 is dev, so it only deploys on pushes to the dev branches, and there are no guarantees about the stability of those workers. If we want a pool deployed for PRs and staging releases on push to the production branches with the latest reviewed and deployed code, we'll want a level 1 prod config for dep.

**Comment by hneiva** [comment]
Linked PR: https://github.com/mozilla-services/cloudops-infra/pull/4133

**Review by JohanLorenzo — APPROVED** [review]
LGTM! Thanks for adding the prod level-1 for regular staging releases 👍

**Comment by jcristau** [comment]
AIUI we can close this, beetmover handles these tasks now.

**Pull request closed without merging**